contents/product-engineers/growth-loops.md

```diff
@@ -1,6 +1,6 @@
 ---
-date: 2022-10-31
-title: How to harness the awesome power of growth loops
+date: 2023-12-15
+title: How to harness the power of growth loops
```

Comment:
Some other headline ideas:

- How successful startups use growth loops (with real examples)
- The growth loops of successful startup (with example)
- How to build a powerful growth loop (with examples)

Reply:
went with "How successful startups use growth loops (with examples)" so I could stay under 60 characters